lib/mixlib/install/generator/powershell/scripts/helpers.ps1.erb

```diff
@@ -150,7 +146,7 @@ function Custom-GetFileHash ($Path, $Algorithm) {
 }
 
 function Get-SHA256Converter {
-  if (Is-FIPS) {
+  if ($(Is-FIPS) -ge 1) {
     New-Object -TypeName Security.Cryptography.SHA256Cng
```

Comment:
I am verifying on nano server also need to verify if any drawback to use `SHA256Cng` rather than it may be support .Net framework 3.5 or above.